public/app/features/explore/ExploreGraphNGPanel.tsx

```diff
@@ -188,24 +189,24 @@ export function ExploreGraphNGPanel({
   );
 }
 
-const getStyles = (theme: GrafanaTheme) => ({
+const getStyles = (theme: GrafanaTheme2) => ({
   timeSeriesDisclaimer: css`
     label: time-series-disclaimer;
     width: 300px;
-    margin: ${theme.spacing.sm} auto;
+    margin: ${theme.spacing(1)} auto;
     padding: 10px 0;
-    border-radius: ${theme.border.radius.md};
+    border-radius: ${theme.spacing(2)};
     text-align: center;
-    background-color: ${theme.colors.bg1};
+    background-color: ${theme.colors.background.primary};
   `,
   disclaimerIcon: css`
     label: disclaimer-icon;
-    color: ${theme.palette.yellow};
-    margin-right: ${theme.spacing.xs};
+    color: ${theme.colors.warning.main};
+    margin-right: ${theme.spacing(1)};
```

Comment:
It was `xs`. Shouldn't it be `{theme.spacing(0.5)};` ?

Reply:
I tried to check the styles that were applied to each html element in the console and it looked like they were the same but i could be wrong. Will check again